ferrocene/doc/release-notes/src/next.rst

```diff
@@ -8,3 +8,28 @@ Next Ferrocene release
 
 This page contains the changes to be introduced in the upcoming Ferrocene
 release.
+
+New features
+------------
+
+The certified subset of the ``core`` library that is certified to IEC-61508 SIL 2 has been significantly expanded.
+For more information, see the :doc:`user-manual:core/index` and :doc:`core-certification:index`.
```

Comment:
user-manual:core just says this: 

<img width="638" height="209" alt="Image" src="https://github.com/user-attachments/assets/cd856551-27b0-48aa-87f9-9484e7a78c00" />

maybe we should link to the API docs directly? that's https://public-docs.ferrocene.dev/main/core/index.html as a URL, idk what it is as a sphinx link.